docs/source/to_do.rst

```diff
@@ -230,6 +230,11 @@ Server
 
 - Improve installation ease and docs.
 
+**Medium**
+
+- Self-test: possible to avoid recreating database structure each time? Makes
+  it very slow.
```

Comment:
Certainly possible with pytest / pytest-django. I haven't checked if there's an equivalent for pyramid. No doubt this would require a lot of work up front and would be good not to have to maintain our own test runner.